wnprc_billing/resources/queries/wnprc_billing/activeTierRates.sql

```diff
@@ -0,0 +1,8 @@
+SELECT
+    tierRateType,
+    tierRate,
+    startDate,
+    endDate,
+    isActive
+FROM wnprc_billing.tierRates
+WHERE isActive = 'Y'
```

Comment:
I'm a little surprised this works, though it's nice. Might be more robust to make this true instead of 'Y'

Reply:
Done!